test/db/cmd/cmd_signature

```diff
@@ -417,49 +403,51 @@ CMDS=<<EOF
 e flirt.sigdb.load.system=false
 e flirt.sigdb.load.home=false
 aaa
-Fs bins/flirt/elf-x86/test.symbols.pat~silence
-pdf @ main
+Fs bins/flirt/elf-x86/test.stripped.pat~silence
+af @ data.004011a5
+pdf @ data.004011a5
```

Comment:
why did you change the function being checked? Can you use that as well in the tests?

Reply:
because i do not have anymore the original binary, so i just changed it with a new one (yours)

Reply:
oh :D ok